news/2018-06-22-cwc-2018-recap-round-of-16.md

```diff
@@ -0,0 +1,59 @@
+
+From this point The Netherlands confidence was shot, and, buoyed by their opening break point victory, Germany reeled off four further game wins in succession to complete the 5-0 sweep. This was some measure of revenge for Germany, who had been soundly beaten by the Dutch in last year’s Semifinals, and they played with neither fear nor mercy in a frightening display of consistency that their future opponents have surely taken notice of.
+
+The Round of 16 concluded with a clinical United States securing a comfortable 5-0 victory over Sweden. Sweden threw quite literally everything in their playbook at this contest, but it merely bounced right off the American armour. Game after game, the Swedish challenge was checked by the Americans, who played with unshakeable consistency throughout and look incredibly strong going forward.
+
+## Upcoming Stage - Quarter Finals
+
+Whilst no further eliminations will occur until next week, our eight Losers' Bracket teams must be feeling the nerves knowing every match is now a fight for survival. On the top side of the draw, the Netherlands look to recover against an inconsistent Japan, whilst Finland seek to continue their impressive run against a struggling Mexico. On the bottom side, Poland and Sweden look to bounce back from major losses, whilst the similarly over-performing France and Norway clash in what could prove to be a gutsy battle of attrition.
+
+If any of the remaining Winners' Bracket teams see this as a comfort net, they will no doubt receive a harsh reality check come the weekend, as the skill level and tournament experience on display is honestly staggering to behold. Germany look to prey upon an inconsistent South Korea in the top half of the draw, joined by a strong Chilean outfit and the ever-dangerous Taiwan.
+
+The bottom half once again resembles a veritable warzone, with four of 2017's top six teams staring each other down for a Semi Final spot. Indonesia first cross blades with Hong Kong in a match oozing with star power and limitless potential, but the match on everybody's must watch list will surely be China locking horns with the United States. This mega match could realistically be a preview of this year's Grand Final, given the incredibly dominant showings by these two juggernauts of the fruit catching world thus far in the tournament, so is absolutely not to be missed!
+
+## Summary
+
+Whether you're a long time fruit-catcher or had never checked out osu!catch before this article, everyone is welcome (and encouraged!) to join us for the Quarter Finals this coming weekend. Follow [@osugame on Twitter](https://twitter.com/osugame) to keep in touch, stay updated, and know when we go live, support your favourite knockout stage team by purchasing a profile banner from the [osu!store](https://osu.ppy.sh/store/listing), and of course, head on over to the [osu!live Twitch channel](https://www.twitch.tv/osulive) to witness all the action as it unfolds.
```

Comment:
there needs to be a stylistic break between the "follow osulive" stuff and the "support your team" one

Reply:
semicolons would do, just don't forget one before "and of course"

Reply:
Why was this sentence so god damn long yuck, split these up into separate sentences